tests/integrations/aws_lambda/test_aws.py

```diff
@@ -22,13 +22,16 @@
 import json
 from sentry_sdk.transport import HttpTransport
 
+FLUSH_EVENT = True
```

Comment:
@rhcarvalho @untitaker defined a global constant to parameterize timeout error to capture events data.

Reply:
yeah that works for me